Singer.API/ClientApp/src/app/modules/shared/components/detailed-user-card/detailed-user-card.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+   selector: 'app-detailed-user-card',
+   templateUrl: './detailed-user-card.component.html',
+   styleUrls: ['./detailed-user-card.component.css'],
+})
+export class DetailedUserCardComponent implements OnInit {
+   @Input() user: any = {};
+   @Input() color: string = '#2a73d4'; //Default color = primary theme color
```

Comment:
Hardcoded color value? Why not in SCSS?

Reply:
Again i do not have acces to SCSS variables in a direct and clean way without using hacks, this color value is just a default for when the user doesn't specify one. Altough you are correct that in this perticular instance it is unnesecary and should be replaced by a CSS Default. And that the Color variable should only be applied with NgStyle when used. I will change this tommorow as well.